### DISCUSSION TITLE
Is a good idea to use next.js with TypeScript?

### DISCUSSION THREAD

**Discussion by wilsonneto-swe** [vercel/next.js#13290] [Help] [answered]
Hi guys again,
sorry for the second questio so quickly... 

But I will startd a project this new monday, and e need to know if is a good idea to use next.js with Type script?

Thanks so much!

**Comment by jamesmosier** [comment, marked as answer]
Yes, it is a great idea. It is also very [easy to get started with](https://nextjs.org/docs/basic-features/typescript)

**Reply by wilsonneto-swe** [reply]
Great, I'll looke at this, thanks so much!
